matrix-sdk-android/src/main/java/im/vector/matrix/android/api/Matrix.kt

```diff
@@ -80,10 +80,12 @@ class Matrix private constructor(context: Context, matrixConfiguration: MatrixCo
                     val matrixConfiguration = (appContext as MatrixConfiguration.Provider).providesMatrixConfiguration()
                     instance = Matrix(appContext, matrixConfiguration)
                 } else {
+                    // TODO: Should set isInit back to false ?
```

Comment:
Yes, probably, but when reaching that, the app will crash, the developer will fix, build and launch the app, so we may ignore